### PR TITLE
Add fixture Kodiaq from chodorenko

### PR DESCRIPTION
**Pull request by dvx76**
From user @chodorenko

### CODE REVIEW

**Review by WebSpider — APPROVED** [review]

**Comment by WebSpider** [comment]
This fixture has capabilities that were recently added, causing tests to fail.

**Comment by prvakt** [comment]
outside_temperature and FLEET_SUPPORTED are missing, please use fixture from below where its fixed
```
name: Kodiaq
description: Fixtures for Kodiaq Style 2023
generation_time: "2024-12-02T18:40:03.948889+00:00"
library_version: 0.13.4
vehicles:
  - id: 0
    device_platform: MBB_ODP
    system_model_id: NS74QZ
    model: Kodiaq
    model_year: "2023"
    trim_level: Style
    software_version:
    capabilities:
      - id: ACCESS
        statuses: []
      - id: DEALER_APPOINTMENT
        statuses: []
      - id: GEO_FENCE
        statuses: []
      - id: HONK_AND_FLASH
        statuses: []
      - id: MEASUREMENTS
        statuses: []
      - id: NEWS
        statuses:
          - LICENSE_MISSING
      - id: PARKING_POSITION
        statuses: []
      - id: ROADSIDE_ASSISTANT
        statuses: []
      - id: SPEED_ALERT
        statuses: []
      - id: STATE
        statuses: []
      - id: TRIP_STATISTICS
        statuses: []
      - id: VEHICLE_HEALTH_INSPECTION
        statuses: []
      - id: WARNING_LIGHTS
        statuses: []
      - id: VEHICLE_WAKE_UP
        statuses: []
      - id: PAY_TO_FUEL
        statuses: []
      - id: CUBIC
        statuses: []
      - id: SUBSCRIPTIONS
        statuses: []
      - id: DESTINATIONS
        statuses:
          - LICENSE_MISSING
          - LICENSE_EXPIRED
      - id: ICE_VEHICLE_RTS
        statuses: []
      - id: FLEET_SUPPORTED
        statuses: []
      - id: LOYALTY_PROGRAM_WORLDWIDE
        statuses: []
      - id: OUTSIDE_TEMPERATURE
        statuses: []
      - id: DIGICERT
        statuses: []
      - id: VEHICLE_HEALTH_WARNINGS
        statuses: []
      - id: PAY_TO_PARK
        statuses: []
      - id: FUEL_STATUS
        statuses: []
      - id: SERVICE_PARTNER
        statuses: []
      - id: ROUTE_IMPORT
        statuses: []
      - id: MISUSE_PROTECTION
        statuses: []
      - id: CARE_AND_INSURANCE
        statuses: []
reports:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: info
    raw:
      '{"vin": "TMOCKAA0AA000000", "name": "Example Car", "workshopModeEnabled":
      false, "licensePlate": "HH AA 1234", "state": "ACTIVATED", "devicePlatform": "MBB_ODP",
      "specification": {"title": "\u0160koda Kodiaq", "manufacturingDate": "2023-02-12",
      "model": "Kodiaq", "modelYear": "2023", "body": "SUV", "trimLevel": "Style", "systemCode":
      "UNKNOWN", "systemModelId": "NS74QZ", "engine": {"type": "TSI", "powerInKW": 140,
      "capacityInLiters": 2.0}, "gearbox": {"type": "A7A"}}, "servicePartner": {"servicePartnerId":
      "DEU11111"}, "renders": [{"url": "https://iprenders.blob.core.windows.net/basens7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}, {"url": "https://ip-modcwp.azureedge.net/modcwpns7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-1920658dayvext_side1080.png?v=638329713820000000",
      "viewPoint": "garage_l", "type": "REAL", "order": 1}, {"url": "https://ip-modcwp.azureedge.net/modcwpns7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-700390dayvext_front1080.png?v=638133583670000000",
      "viewPoint": "main", "type": "REAL", "order": 2}], "compositeRenders": [{"layers":
      [{"url": "https://iprenders.blob.core.windows.net/basens7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}], "viewType": "UNMODIFIED_EXTERIOR_SIDE"},
      {"layers": [{"url": "https://iprenders.blob.core.windows.net/basens7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-19201080dayvext_side1080.png",
      "viewPoint": "exterior_side", "type": "REAL", "order": 0}], "modifications": {"adjustSpaceInPx":
      {"top": -287, "bottom": -200, "left": -138, "right": -93}, "densityIndependentHeight":
      270, "flipHorizontal": false, "anchorTo": "LEFT"}, "viewType": "HOME"}], "capabilities":
      {"capabilities": [{"id": "ACCESS", "statuses": []}, {"id": "DEALER_APPOINTMENT",
      "statuses": []}, {"id": "GEO_FENCE", "statuses": []}, {"id": "HONK_AND_FLASH",
      "statuses": []}, {"id": "MEASUREMENTS", "statuses": []}, {"id": "NEWS", "statuses":
      ["LICENSE_MISSING"]}, {"id": "PARKING_POSITION", "statuses": []}, {"id": "ROADSIDE_ASSISTANT",
      "statuses": []}, {"id": "SPEED_ALERT", "statuses": []}, {"id": "STATE", "statuses":
      []}, {"id": "TRIP_STATISTICS", "statuses": []}, {"id": "VEHICLE_HEALTH_INSPECTION",
      "statuses": []}, {"id": "WARNING_LIGHTS", "statuses": []}, {"id": "VEHICLE_WAKE_UP",
      "statuses": []}, {"id": "PAY_TO_FUEL", "statuses": []}, {"id": "CUBIC", "statuses":
      []}, {"id": "SUBSCRIPTIONS", "statuses": []}, {"id": "DESTINATIONS", "statuses":
      ["LICENSE_MISSING", "LICENSE_EXPIRED"]}, {"id": "ICE_VEHICLE_RTS", "statuses":
      []}, {"id": "FLEET_SUPPORTED", "statuses": []}, {"id": "LOYALTY_PROGRAM_WORLDWIDE",
      "statuses": []}, {"id": "OUTSIDE_TEMPERATURE", "statuses": []}, {"id": "DIGICERT",
      "statuses": []}, {"id": "VEHICLE_HEALTH_WARNINGS", "statuses": []}, {"id": "PAY_TO_PARK",
      "statuses": []}, {"id": "FUEL_STATUS", "statuses": []}, {"id": "SERVICE_PARTNER",
      "statuses": []}, {"id": "ROUTE_IMPORT", "statuses": []}, {"id": "MISUSE_PROTECTION",
      "statuses": []}, {"id": "CARE_AND_INSURANCE", "statuses": []}]}}'
    url: "/v2/garage/vehicles/TMOCKAA0AA000000?connectivityGenerations=MOD1&connectivityGenerations=MOD2&connectivityGenerations=MOD3&connectivityGenerations=MOD4"
    result:
      state: ACTIVATED
      specification:
        body: SUV
        engine:
          power: 140
          type: TSI
          capacity_in_liters: 2
        model: Kodiaq
        title: Škoda Kodiaq
        manufacturing_date: "2023-02-12"
        model_year: "2023"
        system_code: UNKNOWN
        system_model_id: NS74QZ
        battery:
        max_charging_power:
        trim_level: Style
      vin: TMOCKAA0AA000000
      name: Example Car
      capabilities:
        capabilities:
          - id: ACCESS
            statuses: []
          - id: DEALER_APPOINTMENT
            statuses: []
          - id: GEO_FENCE
            statuses: []
          - id: HONK_AND_FLASH
            statuses: []
          - id: MEASUREMENTS
            statuses: []
          - id: NEWS
            statuses:
              - LICENSE_MISSING
          - id: PARKING_POSITION
            statuses: []
          - id: ROADSIDE_ASSISTANT
            statuses: []
          - id: SPEED_ALERT
            statuses: []
          - id: STATE
            statuses: []
          - id: TRIP_STATISTICS
            statuses: []
          - id: VEHICLE_HEALTH_INSPECTION
            statuses: []
          - id: WARNING_LIGHTS
            statuses: []
          - id: VEHICLE_WAKE_UP
            statuses: []
          - id: PAY_TO_FUEL
            statuses: []
          - id: CUBIC
            statuses: []
          - id: SUBSCRIPTIONS
            statuses: []
          - id: DESTINATIONS
            statuses:
              - LICENSE_MISSING
              - LICENSE_EXPIRED
          - id: ICE_VEHICLE_RTS
            statuses: []
          - id: FLEET_SUPPORTED
            statuses: []
          - id: LOYALTY_PROGRAM_WORLDWIDE
            statuses: []
          - id: OUTSIDE_TEMPERATURE
            statuses: []
          - id: DIGICERT
            statuses: []
          - id: VEHICLE_HEALTH_WARNINGS
            statuses: []
          - id: PAY_TO_PARK
            statuses: []
          - id: FUEL_STATUS
            statuses: []
          - id: SERVICE_PARTNER
            statuses: []
          - id: ROUTE_IMPORT
            statuses: []
          - id: MISUSE_PROTECTION
            statuses: []
          - id: CARE_AND_INSURANCE
            statuses: []
      renders:
        - url: https://iprenders.blob.core.windows.net/basens7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-19201080dayvext_side1080.png
          type: REAL
          order: 0
          view_point: exterior_side
        - url: https://ip-modcwp.azureedge.net/modcwpns7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-1920658dayvext_side1080.png?v=638329713820000000
          type: REAL
          order: 1
          view_point: garage_l
        - url: https://ip-modcwp.azureedge.net/modcwpns7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-700390dayvext_front1080.png?v=638133583670000000
          type: REAL
          order: 2
          view_point: main
      device_platform: MBB_ODP
      workshop_mode_enabled: false
      composite_renders:
        - layers:
            - url: https://iprenders.blob.core.windows.net/basens7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
          view_type: UNMODIFIED_EXTERIOR_SIDE
        - layers:
            - url: https://iprenders.blob.core.windows.net/basens7v23200004/5X5XDSLGPW-65-Rvio3t1p5gk9_-8s1bgIP.oW_j3BxMlT-qfP3M.TJG0inswacXCg-B0vtg8.XZU6n1mbPqrJdN-OCJWIYSEALQV-19201080dayvext_side1080.png
              type: REAL
              order: 0
              view_point: exterior_side
          view_type: HOME
      service_partner:
        id: DEU11111
      software_version:
      license_plate: HH AA 1234
      errors:
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: status
    raw:
      '{"overall": {"doorsLocked": "YES", "locked": "YES", "doors": "CLOSED", "windows":
      "CLOSED", "lights": "OFF"}, "detail": {"sunroof": "UNSUPPORTED", "trunk": "CLOSED",
      "bonnet": "CLOSED"}, "renders": {"lightMode": {"oneX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1x&theme=LIGHT",
      "oneAndHalfX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1-5x&theme=LIGHT",
      "twoX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=2x&theme=LIGHT",
      "threeX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=3x&theme=LIGHT"},
      "darkMode": {"oneX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1x&theme=DARK",
      "oneAndHalfX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1-5x&theme=DARK",
      "twoX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=2x&theme=DARK",
      "threeX": "https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=3x&theme=DARK"}},
      "carCapturedTimestamp": "2024-12-02T16:46:09Z"}'
    url: "/v2/vehicle-status/TMOCKAA0AA000000"
    result:
      detail:
        bonnet: CLOSED
        sunroof: UNSUPPORTED
        trunk: CLOSED
      overall:
        doors: CLOSED
        doors_locked: "YES"
        lights: "OFF"
        locked: "YES"
        windows: CLOSED
      renders:
        light_mode:
          one_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1x&theme=LIGHT
          one_and_half_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1-5x&theme=LIGHT
          two_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=2x&theme=LIGHT
          three_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=3x&theme=LIGHT
        dark_mode:
          one_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1x&theme=DARK
          one_and_half_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=1-5x&theme=DARK
          two_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=2x&theme=DARK
          three_x: https://mysmob.api.connect.skoda-auto.cz/api/v2/vehicle-status/render?carType=SUV&vehicleState=1-1-1-1-0-0-0-0-0&lastModifiedAt=1730291332&dimension=3x&theme=DARK
      car_captured_timestamp: "2024-12-02T16:46:09+00:00"
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: air_conditioning
    raw:
      '{"state": "INVALID", "runningRequests": [], "steeringWheelPosition": "LEFT",
      "windowHeatingState": {"front": "INVALID", "rear": "INVALID", "unspecified": "INVALID"},
      "timers": [], "errors": [{"type": "UNAVAILABLE_VEHICLE_INFORMATION", "description":
      "APIs for obtaining seat heating status are not available"}, {"type": "UNAVAILABLE_TIMERS_INFORMATION",
      "description": "APIs for obtaining air conditioning timers are not available"},
      {"type": "UNAVAILABLE_CHARGING_INFORMATION", "description": "APIs for obtaining
      power cable connection/lock information are not available"}]}'
    url: "/v2/air-conditioning/TMOCKAA0AA000000"
    result:
      timers: []
      errors:
        - type: UNAVAILABLE_VEHICLE_INFORMATION
          description: APIs for obtaining seat heating status are not available
        - type: UNAVAILABLE_TIMERS_INFORMATION
          description: APIs for obtaining air conditioning timers are not available
        - type: UNAVAILABLE_CHARGING_INFORMATION
          description:
            APIs for obtaining power cable connection/lock information are
            not available
      state: INVALID
      steering_wheel_position: LEFT
      window_heating_state:
        front: INVALID
        rear: INVALID
        unspecified: INVALID
      car_captured_timestamp:
      air_conditioning_at_unlock:
      charger_connection_state:
      charger_lock_state:
      estimated_date_time_to_reach_target_temperature:
      heater_source:
      seat_heating_activated:
      target_temperature:
      window_heating_enabled:
      air_conditioning_without_external_power:
      outside_temperature:
    error:
  - type: get
    vehicle_id: 0
    success: false
    endpoint: auxiliary_heating
    raw:
    url:
    result:
    error: |
      Traceback (most recent call last):
        File "/usr/local/lib/python3.12/site-packages/myskoda/myskoda.py", line 381, in generate_fixture_report
          result = await self.get_endpoint(vin, endpoint, anonymize=True)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.12/site-packages/myskoda/myskoda.py", line 426, in get_endpoint
          return await method(vin, anonymize=anonymize)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.12/site-packages/myskoda/rest_api.py", line 181, in get_auxiliary_heating
          data=await self._make_get_request(url),
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.12/site-packages/myskoda/rest_api.py", line 104, in _make_get_request
          return await self._make_request(url=url, method="GET")
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "/usr/local/lib/python3.12/site-packages/myskoda/rest_api.py", line 94, in _make_request
          response.raise_for_status()
        File "/usr/local/lib/python3.12/site-packages/aiohttp/client_reqrep.py", line 1158, in raise_for_status
          raise ClientResponseError(
      aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url='https://mysmob.api.connect.skoda-auto.cz/api/v2/air-conditioning/TMOCKAA0AA000000/auxiliary-heating'
  - type: get
    vehicle_id: 0
    success: true
    endpoint: positions
    raw:
      '{"positions": [{"type": "VEHICLE", "gpsCoordinates": {"latitude": 53.470636,
      "longitude": 9.689872}, "address": {"city": "Example City", "street": "Example
      Avenue", "houseNumber": "15", "zipCode": "54321", "countryCode": "DEU"}}], "errors":
      []}'
    url: "/v1/maps/positions?vin=TMOCKAA0AA000000"
    result:
      errors: []
      positions:
        - address:
            country_code: DEU
            zip_code: "54321"
            house_number: "15"
            street: Example Avenue
            city: Example City
            country:
          gps_coordinates:
            latitude: 53.470636
            longitude: 9.689872
          type: VEHICLE
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: health
    raw:
      '{"capturedAt":"2024-12-02T16:46:08Z", "mileageInKm": 37711, "warningLights":
      [{"category": "ASSISTANCE", "defects": []    },{"category": "COMFORT", "defects":
      []}, {"category": "BRAKE", "defects": []}, {"category": "ENGINE", "defects": []},
      {"category": "LIGHTING", "defects": []}, {"category": "TIRE", "defects": []},
      {"category": "OTHER", "defects": []}]}'
    url: "/v1/vehicle-health-report/warning-lights/TMOCKAA0AA000000"
    result:
      warning_lights:
        - category: ASSISTANCE
          defects: []
        - category: COMFORT
          defects: []
        - category: BRAKE
          defects: []
        - category: ENGINE
          defects: []
        - category: LIGHTING
          defects: []
        - category: TIRE
          defects: []
        - category: OTHER
          defects: []
      mileage_in_km: 37711
      captured_at: "2024-12-02T16:46:08+00:00"
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: charging
    raw:
      '{"isVehicleInSavedLocation": false, "settings": {"availableChargeModes": []},
      "errors": [{"type": "CARE_MODE_IS_NOT_AVAILABLE", "description": "Care mode is
      not available."}, {"type": "AUTO_UNLOCK_IS_NOT_AVAILABLE", "description": "Auto
      unlock is not available."}, {"type": "MAX_CHARGE_CURRENT_IS_NOT_AVAILABLE", "description":
      "Maximal charge current is not available."}, {"type": "CHARGE_LIMIT_IS_NOT_AVAILABLE",
      "description": "Charge limit is not available."}]}'
    url: "/v1/charging/TMOCKAA0AA000000"
    result:
      errors:
        - type: CARE_MODE_IS_NOT_AVAILABLE
          description: Care mode is not available.
        - type: AUTO_UNLOCK_IS_NOT_AVAILABLE
          description: Auto unlock is not available.
        - type: MAX_CHARGE_CURRENT_IS_NOT_AVAILABLE
          description: Maximal charge current is not available.
        - type: CHARGE_LIMIT_IS_NOT_AVAILABLE
          description: Charge limit is not available.
      settings:
        available_charge_modes: []
        max_charge_current_ac:
        auto_unlock_plug_when_charged:
        battery_support:
        charging_care_mode:
        preferred_charge_mode:
        target_state_of_charge_in_percent:
      is_vehicle_in_saved_location: false
      car_captured_timestamp:
      status:
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: maintenance
    raw:
      '{"maintenanceReport": {"capturedAt": "2024-12-02T16:46:08Z", "inspectionDueInDays":
      591, "inspectionDueInKm": 22700, "mileageInKm": 37711, "oilServiceDueInDays":
      591, "oilServiceDueInKm": 22700}, "preferredServicePartner": {"name": "Example
      Service Partner", "brand": "C", "partnerNumber": "1111", "id": "DEU11111", "contact":
      {"phone": "+49 1234 567890", "url": "https://example.com", "email": "user@example.com"},
      "address": {"city": "Example City", "street": "Example Avenue", "houseNumber":
      "15", "zipCode": "54321", "countryCode": "DEU"}, "location": {"latitude": 53.470636,
      "longitude": 9.689872}, "openingHours": []}, "predictiveMaintenance": {"setting":
      {"serviceActivated": true, "email": "user@example.com", "phone": "+49 1234 567890"}}}'
    url: "/v3/vehicle-maintenance/vehicles/TMOCKAA0AA000000"
    result:
      maintenance_report:
        captured_at: "2024-12-02T16:46:08+00:00"
        mileage_in_km: 37711
        inspection_due_in_days: 591
        inspection_due_in_km: 22700
        oil_service_due_in_days: 591
        oil_service_due_in_km: 22700
      predictive_maintenance:
        setting:
          email: user@example.com
          service_activated: true
          phone: "+49 1234 567890"
          preferred_channel:
      preferred_service_partner:
        address:
          country_code: DEU
          zip_code: "54321"
          house_number: "15"
          street: Example Avenue
          city: Example City
          country:
        brand: C
        contact:
          email: user@example.com
          phone: "+49 1234 567890"
          url: https://example.com
        id: DEU11111
        location:
          latitude: 53.470636
          longitude: 9.689872
        name: Example Service Partner
        opening_hours: []
        partner_number: "1111"
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: driving_range
    raw:
      '{"carType": "gasoline", "totalRangeInKm": 70, "primaryEngineRange": {"engineType":
      "gasoline", "currentSoCInPercent": 17, "currentFuelLevelInPercent": 17, "remainingRangeInKm":
      70}, "carCapturedTimestamp": "2024-12-02T16:46:09Z"}'
    url: "/v2/vehicle-status/TMOCKAA0AA000000/driving-range"
    result:
      car_captured_timestamp: "2024-12-02T16:46:09+00:00"
      car_type: gasoline
      primary_engine_range:
        engine_type: gasoline
        current_fuel_level_in_percent: 17
        current_soc_in_percent: 17
        remaining_range_in_km: 70
      secondary_engine_range:
      total_range_in_km: 70
      ad_blue_range:
    error:
  - type: get
    vehicle_id: 0
    success: true
    endpoint: trip_statistics
    raw:
      '{"overallMileageInKm": 26, "overallTravelTimeInMin": 81, "overallAverageMileageInKm":
      26, "overallAverageTravelTimeInMin": 81, "overallAverageFuelConsumption": 12.53,
      "overallAverageSpeedInKmph": 20, "vehicleType": "FUEL", "detailedStatistics":
      [{"date": "2024-12-02", "tripIds": ["3620291751", "3621436746", "3622867909"],
      "mileageInKm": 26, "travelTimeInMin": 81, "averageFuelConsumption": 12.53, "averageSpeedInKmph":
      20}, {"date": "2024-12-03"}, {"date": "2024-12-04"}, {"date": "2024-12-05"}, {"date":
      "2024-12-06"}, {"date": "2024-12-07"}, {"date": "2024-12-08"}]}'
    url: "/v1/trip-statistics/TMOCKAA0AA000000?offsetType=week&offset=0&timezone=Europe%2FBerlin"
    result:
      vehicle_type: FUEL
      detailed_statistics:
        - date: "2024-12-02"
          average_fuel_consumption: 12.53
          average_speed_in_kmph: 20
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km: 26
          travel_time_in_min: 81
          trip_ids:
            - 3620291751
            - 3621436746
            - 3622867909
        - date: "2024-12-03"
          average_fuel_consumption:
          average_speed_in_kmph:
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km:
          travel_time_in_min:
          trip_ids:
        - date: "2024-12-04"
          average_fuel_consumption:
          average_speed_in_kmph:
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km:
          travel_time_in_min:
          trip_ids:
        - date: "2024-12-05"
          average_fuel_consumption:
          average_speed_in_kmph:
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km:
          travel_time_in_min:
          trip_ids:
        - date: "2024-12-06"
          average_fuel_consumption:
          average_speed_in_kmph:
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km:
          travel_time_in_min:
          trip_ids:
        - date: "2024-12-07"
          average_fuel_consumption:
          average_speed_in_kmph:
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km:
          travel_time_in_min:
          trip_ids:
        - date: "2024-12-08"
          average_fuel_consumption:
          average_speed_in_kmph:
          average_electric_consumption:
          average_recuperation:
          average_aux_consumption:
          mileage_in_km:
          travel_time_in_min:
          trip_ids:
      overall_average_electric_consumption:
      overall_average_fuel_consumption: 12.53
      overall_average_mileage_in_km: 26
      overall_average_speed_in_kmph: 20
      overall_average_travel_time_in_min: 81
      overall_mileage_in_km: 26
      overall_travel_time_in_min: 81
    error:
```

File: fixtures/Fixtures_for_Kodiaq_Style_2023.yaml [hunk 1]
```diff
@@ -0,0 +1,588 @@
...
+        statuses: []
+      - id: DESTINATIONS
+        statuses:
+          - LICENSE_MISSING
+          - LICENSE_EXPIRED
+      - id: ICE_VEHICLE_RTS
+        statuses: []
+      - id: LOYALTY_PROGRAM_WORLDWIDE
+        statuses: []
+      - id: OUTSIDE_TEMPERATURE
```
Comment (prvakt) [inline fixtures/Fixtures_for_Kodiaq_Style_2023.yaml:58]:
```suggestion
      - id: FLEET_SUPPORTED
        statuses: []
      - id: OUTSIDE_TEMPERATURE
```

File: fixtures/Fixtures_for_Kodiaq_Style_2023.yaml [hunk 2]
```diff
@@ -0,0 +1,588 @@
...
+            statuses: []
+          - id: SUBSCRIPTIONS
+            statuses: []
+          - id: DESTINATIONS
+            statuses:
+              - LICENSE_MISSING
+              - LICENSE_EXPIRED
+          - id: ICE_VEHICLE_RTS
+            statuses: []
+          - id: LOYALTY_PROGRAM_WORLDWIDE
```
Comment (prvakt) [inline fixtures/Fixtures_for_Kodiaq_Style_2023.yaml:177]:
```suggestion
          - id: FLEET_SUPPORTED
            statuses: []
          - id: LOYALTY_PROGRAM_WORLDWIDE
```

File: fixtures/Fixtures_for_Kodiaq_Style_2023.yaml [hunk 3]
```diff
@@ -0,0 +1,588 @@
...
+      air_conditioning_at_unlock:
+      charger_connection_state:
+      charger_lock_state:
+      estimated_date_time_to_reach_target_temperature:
+      heater_source:
+      seat_heating_activated:
+      target_temperature:
+      window_heating_enabled:
+      air_conditioning_without_external_power:
+    error:
```
Comment (prvakt) [inline fixtures/Fixtures_for_Kodiaq_Style_2023.yaml:312]:
```suggestion
          outside_temperature:
    error:
```

**Review by prvakt — CHANGES_REQUESTED** [review]
finally figured it out how to suggest changes heh.. fixture should pass all tests once my changes are applied ;)

File: fixtures/Fixtures_for_Kodiaq_Style_2023.yaml [hunk 4]
```diff
@@ -0,0 +1,588 @@
+---
```
Comment (prvakt) [inline fixtures/Fixtures_for_Kodiaq_Style_2023.yaml:1]: don't know if this is allowed or not, other fixutures doesn't have ```---``` on first line
```suggestion
```

**Review by prvakt — CHANGES_REQUESTED** [review]

**Review by prvakt — APPROVED** [review]

**Comment by dvx76** [comment]
Thanks for the help @prvakt !

**Pull request merged**